modules/front-end/src/app/features/safe/feature-flags/index/index.component.html

```diff
@@ -209,6 +209,15 @@
           </ng-template>
         </nz-form-control>
       </nz-form-item>
+      <nz-form-item>
+        <nz-form-label placeholder="A Small Description for feature flag">
+          Description
+        </nz-form-label>
+        <nz-form-control nzHasFeedback nzErrorTip="Cannot exceed 512 characters">
+          <textarea nz-input formControlName="description" placeholder="Description">
+          </textarea>
+        </nz-form-control>
+      </nz-form-item>
```

Comment:
Great work, thank you @sagar110599 , may I ask you to do one more thing? We need to support i18n for all labels, we are using the official **@angular/localize** package. Can you modify this block of code like this?
```html
      <nz-form-item>
        <nz-form-label i18n="@@common.description">Description</nz-form-label>
        <nz-form-control nzHasFeedback nzErrorTip="Cannot exceed 512 characters" i18n-nzErrorTip="@@ff.description.cannot-exceed-512-characters">
          <textarea nz-input formControlName="description" i18n-placeholder="@@common.description" placeholder="Description">
          </textarea>
        </nz-form-control>
      </nz-form-item>
```

After that, please run: `npm run i18n`, this would generate the i18n resources, don't worry about the translation, we can do it afterwards.

Reply:
Hi @cosmos-explorer , After updating the HTML as suggested and running the command npm run i18n following files got updated which are seen in screenshot. Do i need to commit those as well?? 
![image](https://user-images.githubusercontent.com/46983757/223014561-73c6f93f-9470-4fd8-8ed6-9d637709ab29.png)

Reply:
Yes, those files are the translation files, they need to be committed

Reply:
Updated commit as per suggestions.